DragaliaAPI/Services/Game/SavefileService.cs

```diff
@@ -117,13 +117,24 @@ public async Task Import(LoadIndexData savefile)
             this.playerIdentityService.AccountId
         );
 
+        this.apiContext.ChangeTracker.AutoDetectChangesEnabled = false;
```

Comment:
How come you turned this off?

Reply:
Also increases performance (gained around ~1sec in testing iirc) and nothing in here needs change tracking